components/automate-ui/src/app/pages/project/rules/project-rules.component.ts

```diff
@@ -285,11 +296,15 @@ export class ProjectRulesComponent implements OnInit, OnDestroy {
   }
 
   public handleNameInput(event: KeyboardEvent): void {
-    if (!this.modifyID && !this.isNavigationKey(event)) {
+    if (!this.modifyID && !this.isNavigationKey(event) && !this.editingRule) {
```

Comment:
Hrm I wish we could put more logic like this into the formgroup itself; instead of handling it in the component's state... 🤔